### PR TITLE
Fix/flaky tests issues template skip 2

### PR DESCRIPTION
**Pull request by gauthierpetetin**
## **Description**

Tiny adjustments to align with Extension.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by chrisleewilcox** [comment]
@gauthierpetetin please review merge conflict.

**Comment by gauthierpetetin** [comment]
@chrisleewilcox yes, conflicts are now resolved

File: .github/guidelines/LABELING_GUIDELINES.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Once PR has been tested by QA (only if the PR was labeled with `needs-qa`):
 - **QA Passed**: If the PR was labeled with `needs-qa`, this label must be added once QA has signed off
 
 ### Optional labels:
-- **regression-develop**: This label can manually be added to a bug report issue at the time of its creation if the bug is present on development branch (i.e. `main`), but is not yet released in production.
+- **regression-develop**: This label can manually be added to a bug report issue at the time of its creation if the bug is present on the development branch, i.e., `main`, but is not yet released in production.
```
Comment (legobeat) [inline .github/guidelines/LABELING_GUIDELINES.md:25]:
```suggestion
- **regression-develop**: This label can manually be added to a bug report issue at the time of its creation if the bug is present on the development branch, i.e. `main`, but is not yet released in production.
```

that's a comma too many, no?

Comment (gauthierpetetin) [reply .github/guidelines/LABELING_GUIDELINES.md:25]: I don't know exactly. Actually that's a suggestion made by @davidmurdoch here: https://github.com/MetaMask/metamask-extension/pull/23857#discussion_r1569413958

**Review by MarioAslau — APPROVED** [review]

**Pull request merged**